libkineto/src/output_json.cpp

```diff
@@ -348,12 +348,18 @@ void ChromeTraceLogger::handleActivity(const libkineto::ITraceActivity& op) {
     duration += 2; // Still need it to end at the original point rounded up.
   }
 
+  int external_id = 0;
+  if (op.linkedActivity()) {
+    external_id = op.linkedActivity()->correlationId();
+  } else if (op.type() == libkineto::ActivityType::CPU_OP ||
```

Comment:
These types align with https://github.com/pytorch/pytorch/blob/main/torch/csrc/profiler/collection.cpp#L1023-L1026

Reply:
I think logically this makes sense although I was wondering if instead of having this whitelist of events that set external_id = op.correlationId(), can we just have a blacklist of events where we want to skip (ie. CUDA_RUNTIME/CUDA_KERNEL events). I ask this because there could be a type that we are missing and I would rather have extra information (even if wrong) rather than have a trace missing information.

Reply:
Awesome, thanks!